DEPLOY.md

```diff
@@ -76,6 +76,10 @@ parameters:
   iamEndpoint: <IAM API server URL, e.g. http://iam.landemo1.cloudian.eu:16080>
   storagePolicyId: <Storage Policy ID - or omit line to use default storage policy> - greenfield only
   iamPolicy: <IAM policy document (JSON string) for users of this bucket - omit to use default IAM policy (read+write bucket)>
+  createBucketUser: <optional - "yes" or "no", default is "yes">
+  bucketClaimUserSecretName: <A separate secret holding credentials object bucket claim>
+  bucketClaimUserSecretNamespace: <optional if createBucketUser is "yes". A separate secret holding credentials object bucket claim>
```

Comment:
The 2nd sentence looks pasted from SecretName above?  Is this because the two items are a pair?  If so, should they be documented as such, with bucketClaimUserSecretNamespace saying "see bucketClaimUserSecretName"?  Either way, I'm slightly confused...